packages/teleport-plugin-module-generator-angular/src/index.ts

```diff
@@ -0,0 +1,127 @@
+import * as types from '@babel/types'
+import { extractRoutes } from '@teleporthq/teleport-shared/dist/cjs/utils/uidl-utils'
+import { ComponentPluginFactory, ComponentPlugin } from '@teleporthq/teleport-types'
+import { CHUNK_TYPE, FILE_TYPE } from '@teleporthq/teleport-shared/dist/cjs/constants'
+import { camelCaseToDashCase } from '@teleporthq/teleport-shared/dist/cjs/utils/string-utils'
+
+import {
+  createRoutesAST,
+  createPageRouteAST,
+  createExportModuleAST,
+  constructLocalDependency,
+  createRootModuleDecorator,
+  constructComponentDependency,
+  createComponentModuleDecorator,
+  createPageModuleModuleDecorator,
+  constructRouteForComponentsModule,
```

Comment:
nice arrangement, @anamariaoros would be proud 🤣

Reply:
I thought i am the only one, who does this 😂